packages/react-union-scripts/scripts/lib/__tests__/utils-test.js

```diff
@@ -145,36 +144,18 @@ describe('utils', () => {
 	describe('mergeWhen', () => {
 		it('should fire function with args when condition is met and return res of the function', () => {
 			const fn = jest.fn(() => 'res');
-			const res = mergeWhen(true, fn, 'a', 'b');
+			const res = mergeWhen(true, fn, ['a', 'b']);
 			expect(res).toEqual('res');
 			expect(fn).toHaveBeenCalledWith('a', 'b');
 		});
 		it('should not fire function when condition is unmet and return {}', () => {
 			const fn = jest.fn();
-			const res = mergeWhen(false, fn, 'a', 'b');
+			const res = mergeWhen(false, fn, ['a', 'b']);
 			expect(res).toEqual({});
 			expect(fn).not.toHaveBeenCalled();
 		});
 	});
-	describe('resolveAsyncSuffix', () => {
-		it('should return unchanged regexp when is regexp', () => {
-			const reg = /ahoj/;
-			expect(resolveAsyncSuffix(reg)).toBe(reg);
-		});
-		it('should return regexp constructed from string', () => {
-			const expected = R_.constructRegExp('.ahoj.js$', 'i');
-			expect(resolveAsyncSuffix('ahoj')).toEqual(expected);
-		});
-		it('should return regexp constructed from array of strings', () => {
-			const expected = R_.constructRegExp('.(ahoj|ahoj|ahoj).js$', 'i');
-			expect(resolveAsyncSuffix(['ahoj', 'ahoj', 'ahoj'])).toEqual(expected);
-		});
-		it('should throw error when is something different', () => {
-			expect(() => {
-				resolveAsyncSuffix({}).toThrowError();
-			});
-		});
-	});
+
```

Comment:
I am really sad about dropping that feature.

Reply:
I understand, but it is simply not unnecessary. This is pretty much the same thing as dropping SCSS loader – sure, it might useful, but we have better solutions and keeping this one would be a burden to the codebase.